.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json.schemastore.org/prettierrc",
+  "arrowParens": "avoid",
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "useTabs": false
+}
```

Comment:
Is this file useful ? I think we can remove it from PR.

Reply:
I find this file useful for the project as it helps a new contributor to work on existing code without breaking code style. I'll remove this file as it has nothing to do with PR subject.

A new PR can be created only to add this file if that makes sense to you.



This file is useful for the project to help a new contributor working on existing code without breaking source indent/format. But

Reply:
Ok let's keep it here then.